README.md

```diff
@@ -151,12 +125,17 @@ perf/perfConsumer
 ```shell
 # For openSSL
 brew install openssl
+
+# For x86_64 macOS
 export OPENSSL_INCLUDE_DIR=/usr/local/opt/openssl/include/
 export OPENSSL_ROOT_DIR=/usr/local/opt/openssl/
 
+# For arm64 (Apple Silicon) macOS
+export OPENSSL_INCLUDE_DIR=/opt/homebrew/opt/openssl/include/
+export OPENSSL_ROOT_DIR=/opt/homebrew/opt/openssl/
+
 # For Protobuf
-brew install protobuf boost boost-python log4cxx jsoncpp
-// If you are using python3, you need to install boost-python3
+brew install protobuf boost boost-python3 log4cxx jsoncpp
```

Comment:
`jsoncpp` is not needed.

In addition, I think we can remove all `log4cxx` references because it's disabled by default. And I think we can just remove `log4cxx` in future.